src/Monolog/Formatter/HtmlFormatter.php

```diff
@@ -136,4 +133,9 @@ protected function convertToString($data): string
 
         return Utils::jsonEncode($data, JSON_PRETTY_PRINT | Utils::DEFAULT_JSON_FLAGS, true);
     }
+    
+    protected function convertToSpecialChars($data): string
```

Comment:
why making this a protected method ?

Reply:
I forgot to fix it, after copying from the above function, now I'm fixing it.

Reply:
I don't think public is better :) Probably should be private?